cinema/views.py

```diff
@@ -127,6 +123,29 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "title",
+                type={"type": "string", "items": {"type": "str"}},
+                description="Filter by movie title (ex. ?title=Interstellar)",
+            ),
+            OpenApiParameter(
+                "genres",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by genres id (ex. ?genres=4,5)",
+            ),
+            OpenApiParameter(
+                "actors",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by actors id (ex. ?genres=2,5)",
```

Comment:
The description for the 'actors' parameter in the OpenApiParameter should be updated to 'Filter by actors id (ex. ?actors=2,5)' to match the parameter name.